hls4ml/model/hls_layers.py

```diff
@@ -1842,6 +1846,7 @@ def _get_transforms_config(self, params):
     'Resize'                 : Resize,
     'UpSampling2D'           : Resize,
     'Transpose'              : Transpose,
+    'Permute'                : Transpose,
```

Comment:
You don't need this since you put `Transpose` as the `class_name` attribute in the handler function.

Reply:
Agreed this can be removed.